### PR TITLE
Add script options and flag for starting container during deployment

### PR DESCRIPTION
**Pull request by harshalchaudhari35**
This pull request addresses the issue #1840 

+ What existing problem does this PR solve?
Inline with some previous discussion/comments from the maintainers, have considered to add flag and options to the bash script to provide fine grained control to the user, i.e to start timesketch container and create a new user.

p.s. hope I have not over-engineered the solution

**Checks**
- [X] Tested locally for the conditional parts of the bash script

### CODE REVIEW

**Review by berggren — CHANGES_REQUESTED** [review]
Thanks for contributing! Looks good. PTAL

File: contrib/deploy_timesketch.sh [hunk 1]
```diff
@@ -118,14 +124,41 @@ ln -s ./config.env ./timesketch/.env
...
+fi
+
+read -p "Would you like to create a new timesketch user? [Y/n] (default:no)" CREATE_USER
+
+if [ "$CREATE_USER" != "${CREATE_USER#[Yy]}" ] ;then
+  read -p "Please provide a new username: " NEWUSERNAME
+
+  if [ ! -z "$NEWUSERNAME" ] ;then
+    until [ "`docker inspect -f {{.State.Health.Status}} timesketch-web`"=="healthy" ]; do
+      sleep 0.1;
```
[the remark this answers is not in the document]
Comment (harshalchaudhari35) [reply contrib/deploy_timesketch.sh:159]: fair point :)

File: contrib/deploy_timesketch.sh [hunk 2]
```diff
@@ -15,6 +15,12 @@
 
 set -e
 
+START_CNTR=
```
[the remark this answers is not in the document]
Comment (harshalchaudhari35) [reply contrib/deploy_timesketch.sh]: done

**Comment by harshalchaudhari35** [comment]
@berggren gentle nudge on this one, think I've addressed the review comments.

**Review by berggren — APPROVED** [review]
LGTM. Thanks for contributing!

**Pull request merged**